_profiles/work.html

```diff
@@ -0,0 +1,48 @@
+---
+#
+# The human-friendly name of the profile. Whenever you're asked for the profile
+# ID, it refers to the name of the profile file — in this case, "default".
+#
+name: "Work"
+
+#
+# The default profile will be the one shown when accessing the root URL of your
+# SpeedTracker site.
+#
+default: false
+
+#
+# If you want tests to be repeated automatically, you can an interval (in hours)
+# with a minimum value of 12, as only two scheduled tests per day are allowed.
+# To disabled repeated tests, simply remove this property from the profile.
+#
+interval: 12
+
+#
+# WebPageTest test parameters
+#
+parameters:
+  connectivity: "Cable"
+  location: "London_EC2"
+  runs: 1
+  url: "https://wellcomecollection.org/works/v9g6hmpw?query=david&cohort=platformTeam"
```

Comment:
Why?

Reply:
Good question/spot. Originally, on `/works`, this ensured that we would always hit the origin, so we have a worst-case measure. But this doesn't hold up here. If we want to continue to hit the origin for this page, we'll need to find another way of doing it. Suggestions?

Reply:
It's only images no?

At the moment this isn't necessarily breaking cache, but probably is as that URL won't be hit / cached that often.

We could just leave it.